plutus-tx-plugin/src/PlutusTx/Plugin.hs

```diff
@@ -215,8 +215,11 @@ mkPluginPass opts = GHC.CoreDoPluginPass "Core to PLC" $ \ guts -> do
 type PluginM uni fun = ReaderT PluginCtx (ExceptT (CompileError uni fun Ann) GHC.CoreM)
 
 -- | Runs the plugin monad in a given context; throws a Ghc.Exception when compilation fails.
-runPluginM :: (PLC.Pretty (PLC.SomeTypeIn uni), PLC.Closed uni, PLC.Everywhere uni PLC.PrettyConst, PP.Pretty fun)
-           => PluginCtx -> PluginM uni fun a -> GHC.CoreM a
+runPluginM
+    :: ( PLC.Pretty (PLC.SomeTypeIn uni), PLC.PrettyReadable (PLC.SomeTypeIn uni)
+       , PLC.Closed uni, PLC.Everywhere uni PLC.PrettyConst, PP.Pretty fun
```

Comment:
It really looks like we should `PrettyBuiltins uni fun` or something, given the amount of repetition.

Reply:
I decided to bite the bullet and perform the refactoring, but I'll do it in a separate PR.